access/src/main/webapp/WEB-INF/jsp/fullRecord.jsp

```diff
@@ -20,9 +20,10 @@
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <%@ taglib prefix="cdr" uri="http://cdr.lib.unc.edu/cdrUI" %>
 
-<c:if test="${not empty briefObject && cdr:contains(briefObject.status, 'Deleted') || cdr:contains(briefObject.status, 'Parent Deleted')}">
-	<c:set var="isDeleted" value="deleted" scope="page"/>
+<c:if test="${markedForDeletion}">
+	<c:set var="isDeleted" value="deleted" scope="session"/>
```

Comment:
i think "session" will stick around past the current request. If you need a broader scope than "page", i think you can use "request" to avoid some potentially weird interference